### ISSUE TITLE
[Bug] <Sen Manga>: Titles with hyphens `-` are missing when viewed from Tachiyomi

### ISSUE THREAD

**Issue by illiiilli** [tachiyomiorg/tachiyomi-extensions#7524]
## Device information
- Tachiyomi version: 0.11.1 (2021/06/01 7:39)
- Android version: 11

## Source information
- Name: Sen Manga
- Extension version: 1.2.5

## Steps to reproduce
1. ブラウズ (Browse) → ソース (Source) → Sen Manga → 絞り込む (Sort) → Sort By: Last Update
2. Compare the output of Step 1 with the page on [Sen Manga](https://raw.senmanga.com/). 
    This problem does not appear when viewed with WebView.

### Expected behavior

Usually when one browses Sen Manga with a normal browser,
both titles with hyphens `-` as the word separator,
and titles with spaces ` ` as the word separator appear.

In other words,
something resembling ![this Sen Manga when viewed using WebView](https://i.imgur.com/pb47scb.png).

### Actual behavior

However,
when browsing with Tachiyomi,
only titles with spaces ` ` as the word separator appear.

See this image: ![Sen Manga when viewed using Tachiyomi](https://i.imgur.com/8Qb50WE.png)

When you compare the two,
Ao-no-Hako,
Kowloons-Ball-Parade,
Witch-Watch,
and
Jujutsu-Kaisen
are all missing.

## Other details

N/A

**Comment by nar1n** [comment]
This is because latest view is scraped from https://raw.senmanga.com/directory/last_update
